spec.md

```diff
@@ -2837,7 +3275,10 @@ statement = "wire" , id , ":" , type , [ info ]
           | "stop(" , expr , "," , expr , "," , int , ")" , [ info ]
           | "printf(" , expr , "," , expr , "," , string ,
             { expr } , ")" , [ ":" , id ] , [ info ]
-          | "skip" , [ info ] ;
+          | "skip" , [ info ]
+          | "export" , ref_expr , "as" , static_reference , [ info ]
+          | "forward" , static_reference , "as" , static_reference , [ info ]
+          | force_release ;
```

Comment:
Nit: for consistency, it may be useful to move the `[ info ]` specification here and not put it in `force_release`.